filebeat/module/elasticsearch/slowlog/config/slowlog.yml

```diff
@@ -6,7 +6,7 @@ paths:
 exclude_files: [".gz$"]
 
 multiline:
-  pattern: '^\[?[0-9]{4}-[0-9]{2}-[0-9]{2}'
+  pattern: '^(\[?[0-9]{4}-[0-9]{2}-[0-9]{2}|{)'
```

Comment:
This is kind of interesting. In the event for json it's not a multiline anymore but here we still need the multiline config 🤔 

Good for now, jus thought I mention it.